sampleproject/components/recursive.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, Dict
+
+from django_components import Component, register, types
+
+
+@register("recursive")
+class Recursive(Component):
```

Comment:
I also included the recursive component in sampleproject, to showcase what the library can do.